documentation/slate/source/includes/apis.md

```diff
@@ -1028,6 +1028,60 @@ nop      | data_freshness   | enum                           | Define the freshn
 <a name="departures"></a>Departures
 -----------------------------------
 
+``` shell
+
+#using "headers"
+$ curl 'https://api.navitia.io/v1/coverage/sandbox/lines/line:RAT:M1/departures?from_datetime=20160615T1337' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'
+
+HTTP/1.1 200 OK
```

Comment:
You should add "#Request" and "#Response" comments, as it is done in the other example:
`#Request`
`$ curl 'https://api.navitia.io/v1/coverage/sandbox/lines/line:RAT:M1/departures?from_datetime=20160615T1337' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'`

`#Response`
`HTTP/1.1 200 OK`